<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1452</width>
    <height>706</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calibration </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1411</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="importMeasurementTab">
     <attribute name="title">
      <string>Import Measurement</string>
     </attribute>
     <widget class="QPushButton" name="applyParametersButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>430</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply parameters to preview</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>611</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Import the data</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="measurementLineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="measurementBrowseButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="flipDataCheckBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>64</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="dataTypeComboBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string>SSRM</string>
      </property>
      <item>
       <property name="text">
        <string>SSRM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
      </item>
     </widget>
     <widget class="QSlider" name="leftBorderSlider">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>285</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSlider" name="rightBorderSlider">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>370</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="denominationLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="dataPathLabel">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>40</y>
        <width>741</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="plotWidget" native="true">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>50</y>
        <width>851</width>
        <height>581</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>831</width>
         <height>561</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
     <widget class="QLabel" name="leftMinLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>310</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="leftMaxLabel">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>310</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="rightMaxLabel">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>400</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="rightMinLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>400</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="rightSliderValueLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="leftSliderValueLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>160</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Data File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Data Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="DataDenominationLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Data Denomination</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Flip Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Left Border</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rightBorderValueLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Right Border</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Select Calibration File</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Alignment v2.0</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Fitpoints</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Calibration</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1452</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
